x-pack/plugins/rollup/public/crud_app/services/query_params.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export function extractQueryParams(queryString) {
```

Comment:
Correct, react-router only supplies the search string, without parsing it into the individual query keys and values (https://github.com/ReactTraining/react-router/issues/4410). We could use a lib instead of hand-rolling this, but I think I want to avoid introducing yet another dependency.